ionic_user_interface/src/components/wall-image-viewer/Canvas.vue

```diff
@@ -155,6 +155,18 @@ export default defineComponent({
     const isLoading = ref(false);
     const routeMakingStep = ref(0);
 
+    const computeDimensions = (heightToWidthRatio: number) => {
```

Comment:
The problem is not as bad for `Canvas.vue` but more for `ViewRoute.vue`

Reply:
Yea, the only way i can think of to fix is to resize the image based on height, then resize the rest of the elements based on the new resized image width